spec/features/course/experience_points_disbursement_spec.rb

```diff
@@ -17,9 +22,23 @@
     context 'As a Course Teaching Assistant' do
       let(:user) { course_teaching_assistant.user }
 
+      scenario 'I can filter students by group' do
+        approved_course_students
+        visit disburse_experience_points_course_users_path(course)
+        approved_course_students.each do |student|
+          expect(page).to have_content_tag_for(student)
+        end
+
+        click_link group.name
+        expect(page).not_to have_content_tag_for(ungrouped_course_student)
+        grouped_course_students.each do |student|
+          expect(page).to have_content_tag_for(student)
+        end
+      end
+
       scenario 'I can disburse experience points' do
         unapproved_course_student
-        unawarded_student, awarded_student, awarded_zero_student = course_students
+        unawarded_student, awarded_student, awarded_zero_student = approved_course_students
```

Comment:
@weiqingtoh Do you mean that it seems confusing because we create `course_user`s via the `course_group_user` factory in `let(:grouped_course_students)`? Could you explain a bit more?

@allenwq Sorry, don't really understand. Do you mind explaining what in `unawarded_student, awarded_student, awarded_zero_student` is confusing? What impression do you get when I use those names?

Agree with @weiqingtoh that we can polish this up in another PR.